lark/visitors.py

```diff
@@ -15,9 +15,16 @@ class Discard(Exception):
     """
     pass
 
-# Transformers
+class Inline(object):
+    """When returning an Inline instance in a transformer callback,
```

Comment:
Question: Will this make sure that it is added to the docs?

Anyway, an example would be nice, as well as pointing out to use it `Inline(a, b, c)` and not `Inline((a,b,c))`.

And it should be added to the stubs.